modules/ROOT/pages/use-log4j-appender.adoc

```diff
@@ -21,7 +21,17 @@ You can use only asynchronous log appenders.
 
 == Enabling Log4j 
 
-Custom Log4j configurations are enabled by default. When you install Runtime Fabric, the parameter is enabled in `Values.yaml`. Even though the value is set to `true`, you must add an appender. To disable Log4j, set the value to `false`.
+Custom Log4j configurations are disabled by default. 
+
+To enable custom Log4j configurations perform the following steps:
+
+. From inside your cluster, run:
++
+----
+kubectl -nrtf patch secret custom-properties -p '{"data":{"CUSTOM_LOG4J_ENABLED":"dHJ1ZQo="}}'
+----
+. Restart the Runtime Fabric agent pod. 
```

Comment:
It would be helpful to have examples for all commands, i.e.:
```sh
kubectl --namespace rtf delete pod -lapp=agent
```
to restart the pod, etc.